aws/extensions/computetask_linux_docker_compose/extension.ftl

```diff
@@ -0,0 +1,60 @@
+[#ftl]
+
+[@addExtension
+    id="computetask_linux_docker_compose"
+    aliases=[
+        "_computetask_linux_docker_compose"
+    ]
+    description=[
+        "Installs the docker compose"
+    ]
+    supportedTypes=[
+        EC2_COMPONENT_TYPE,
+        COMPUTECLUSTER_COMPONENT_TYPE
+    ]
+    scopes=[
+        COMPUTETASK_EXTENSION_SCOPE
+    ]
+/]
+
+[#macro shared_extension_computetask_linux_docker_compose_deployment_computetask occurrence ]
+
+    [#local version = "1.29.1"]
+
+    [#local commands = {}]
+    [#local commands += {
+        "01DownloadDockerCompose" : {
+            "command" : "curl -L \"https://github.com/docker/compose/releases/download/${version}/docker-compose-$(uname -s)-$(uname -m)\" -o /usr/local/bin/docker-compose",
+            "ignoreErrors" : false
+        },
+        "02ApplyExecutablePermission" : {
+            "command" : "chmod +x /usr/local/bin/docker-compose",
+            "ignoreErrors" : false
+        },
+        "02ApplyExecutablePermission" : {
+            "command" : "chmod +x /usr/local/bin/docker-compose",
+            "ignoreErrors" : false
+        },
```

Comment:
Looks like a duplicate of the same script

Reply:
removed